recipe/build.sh

```diff
@@ -6,10 +6,10 @@ set -vexu -o pipefail
 
 pushd nim
 # inject compilers
-echo "gcc.exe = \"$CC\"" >> config/nim.cfg
-echo "gcc.linkerexe = \"$CC\"" >> config/nim.cfg
-echo "clang.exe = \"$CC\"" >> config/nim.cfg
-echo "clang.linkerexe = \"$CC\"" >> config/nim.cfg
+echo "gcc.exe = \"$(basename $CC)\"" >> config/nim.cfg
+echo "gcc.linkerexe = \"$(basename $CC)\"" >> config/nim.cfg
```

Comment:
Shoudln't this be LD ?

Reply:
I've asked around, and we're pretty sure we want to link using gcc, rather than using ld. Anway, the crash is at compile-time.